__tests__/app.test.js

```diff
@@ -129,30 +150,42 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.message).toBe('Invalid ID')
         })
     })
-    test('should return a 406 Not acceptable if the object passed is incorrectly formatted',()=>{
+    test('should return a 400 Bad Request if the object passed is incorrectly formatted',()=>{
```

Comment:
Not sure if this test overlaps with the next one with a missing property. Also, the comment does not look badly formatted